test/e2e/pages/vl-side-sheet.page.js

```diff
@@ -44,6 +45,23 @@ class VlSideSheetPage extends Page {
     	return this._getSideSheet('#vl-side-sheet-left');
     }
 
+    async clickOpenSideSheetButton() {
+        const button = await this._getButton('#side-sheet-with-datepicker-open-button');
+        await button.click();
+    }
+
+    async getSidesheetDatepicker() {
+        return this._getDatepicker('#vl-datepicker');
+    }
+
+    async _getDatepicker(selector) {
+        await this.driver.wait(async () => {
+            let el = await this.driver.findElement(By.css(selector));
+            return el.isDisplayed();
+        }, 3000);
+        return new VlDatepicker(this.driver, selector);
+    }
```

Comment:
Kunnen we dit niet beter in de constructor zetten van `VlDatepicker`? Vermoedelijk moeten nu al de afnemers dit voorzien?

Reply:
Kan wel zijn dat er al een element wordt gedefinieerd, maar die initieel hidden is.

Reply:
De code kon verwijderd worden.